chainerx_cc/chainerx/routines/math.h

```diff
@@ -117,30 +117,8 @@ Array IsFinite(const Array& x);
 
 Array Tanh(const Array& x);
```

Comment:
Isn't `Tanh` to be included in the "hyperbolic" routines and kernels?

Reply:
Moved to `hyperbolic.h`